README.md

```diff
@@ -210,15 +220,15 @@ Once you're there, ask your solution a question. Like before, you question shoul
 
 Most of the Azure resources deployed in the prior steps will incur ongoing charges unless removed. Also a few of the resources deployed go into a soft delete status. It's best to purge those once you're done exploring, Key Vault is given as an example here. Azure OpenAI and Azure Machine Learning Workspaces are others that should be purged.
 
+> **_NOTE:_** To purge the AI services resources, `Microsoft.CognitiveServices/locations/resourceGroups/deletedAccounts/delete` permission must be given to the user on the subscription scope where the resources are deployed. A role assignment at the resource or resource group scopes will be insufficient to access the purge functionality. See the [learn documentation](https://learn.microsoft.com/azure/ai-services/recover-purge-resources?tabs=azure-cli#purge-a-deleted-resource) for additional details.
+
```

Comment:
I don't think this note is relevant in this scenario.  We create resource directly as part of this, and the prereqs already cover this operation.  Am I missing something?

 ```suggestion
```